src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -297,7 +163,9 @@ def _in_process_execute(self, command):
         try:
             # issue: stderr cannot be redirect in this form, as a result some failure information
             # is lost when command fails.
-            self.exit_code = cli_main(shlex.split(command), file=output_buffer) or 0
+            cmd_parts = [part.replace('"', '')
```

Comment:
Be careful of this replacing logic it converts `"\""` to `\`. What you actually try to do is `strip` the double quotes.